lib/Conversion/TorchToLinalg/TorchToLinalg.cpp

```diff
@@ -1131,6 +1131,26 @@ class ConvertAtenBmmOp : public OpConversionPattern<AtenBmmOp> {
 };
 } // namespace
 
+namespace {
+class ConvertAtenDropoutOp : public OpConversionPattern<AtenDropoutOp> {
+public:
+  using OpConversionPattern::OpConversionPattern;
+  LogicalResult
+  matchAndRewrite(AtenDropoutOp op, OpAdaptor adaptor,
+                  ConversionPatternRewriter &rewriter) const override {
+    if (failed(verifyLinalgCompatibleTypes(op, rewriter)))
+      return failure();
+
```

Comment:
nit: It's safer to verify the `train` operation is constant False before continue.